wake/demo.wake

```diff
@@ -116,7 +116,7 @@ global def capctlDUT =
 
 global def demoDUT =
   def name = "demoDUT"
-  def blocks = pwmBlock, capctlBlock, pioBlock, Nil   #@tom add capcrlBlock
+  def blocks = capctlBlock, pioBlock, Nil   #@tom add capcrlBlock
```

Comment:
since the `capctlBlock` includes a `pwmBlock` already there should not be another `pwmBlock` added to the `demoDUT`.